src/tests/test_rtc_data_models_aec.py

```diff
@@ -487,39 +488,42 @@ def test_validate_datetime_tz(self) -> None:
 
         expected_validation_errors = [
             {
-                'loc': ('fecha_firma_dt',),
-                'msg': 'Value must be a timezone-aware datetime object.',
                 'type': 'value_error',
-            },
+                'loc': ['fecha_firma_dt'],
+                'msg': 'Value error, Value must be a timezone-aware datetime object.',
+                'input': '2019-04-05T12:57:32',
+                'ctx': {'error': 'Value must be a timezone-aware datetime object.'},
+                'url': pydantic_utils.PYDANTIC_VALUE_ERROR_URL,
+            }
         ]
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 fecha_firma_dt=datetime(2019, 4, 5, 12, 57, 32),
             )
 
-        validation_errors = assert_raises_cm.exception.errors()
-        self.assertEqual(len(validation_errors), len(expected_validation_errors))
-        for expected_validation_error in expected_validation_errors:
-            self.assertIn(expected_validation_error, validation_errors)
+        validation_errors = json.loads(assert_raises_cm.exception.json())
+        self.assertEqual(validation_errors, expected_validation_errors)
```

Comment:
(⚠️ This review comment applies to all similar cases.)

```suggestion
        validation_errors = assert_raises_cm.exception.errors(include_url=False)
        self.assertEqual(len(validation_errors), len(expected_validation_errors))
        for expected_validation_error in expected_validation_errors:
            self.assertIn(expected_validation_error, validation_errors)
```

Reply:
Most of cases I changed it to check the whole error insteand of an item becasue is expected to only get the tested error

Should we only check if the error we expect is there and ignore any other errors?

Reply:
> Most of cases I changed it to check the whole error insteand of an item becasue is expected to only get the tested error

It's easier to use the same testing code for single and multiple errors.

> Should we only check if the error we expect is there and ignore any other errors?

We should test not only for expected errors, but also for unexpected errors. For example, if some code should cause two errors, we should check that those two errors are there, and also that there aren't any other errors that we didn't expect.